### PR TITLE
Add <StickyNav> to features page

### PR DESCRIPTION
**Pull request by Ken-Lo**
Use `<StickyNav>` component on Features page (updated version).

### CODE REVIEW

**Comment by Ken-Lo** [comment]
Hey @jamesshedden
I think we forgot to change over to the `<StickyNav>` component on the Features page. Mind taking a quick look?:) This is the updated version with en, fr and de.
Thanks!

**Comment by Ken-Lo** [comment]
Bump @jamesshedden

File: app/pages/features/features.de.js [hunk 1]
```diff
@@ -36,43 +37,39 @@ export default class FeaturesDe extends React.Component {
               </div>
             </div>
           </div>
-          <div className='sticky-nav'>
-            <div ng-gc-sticky-nav
```
Comment (jamesshedden) [inline app/pages/features/features.de.js]: `ng-gc-sticky-nav` attribute no longer needed, it's an artefact of our previous Angular-based setup

File: app/pages/features/features.de.js [hunk 2]
```diff
@@ -36,43 +37,39 @@ export default class FeaturesDe extends React.Component {
...
-              className='sticky-nav__inner u-background-light-gray u-text-heading u-text-xxs u-text-light u-text-no-smoothing'>
-              <div className='site-container u-padding-Vm'>
-                <ul className='u-pull-start u-margin-Txxs'>
-                  <li className='sticky-nav__item'>
-                    <a href='#overview' du-scrollspy ng-gc-smooth-scroll id='track-sticky-nav-overview' className='sticky-nav__link'>
```
Comment (jamesshedden) [inline app/pages/features/features.de.js]: `du-scrollspy ng-gc-smooth-scroll` no longer needed (applies to all elements with this on)

File: app/pages/features/features.de.js [hunk 3]
```diff
@@ -36,43 +37,39 @@ export default class FeaturesDe extends React.Component {
...
-                  }>
-                  <Message pointer='cta.pro' />
-                </Link>
-              </div>
-              <hr className='u-margin-An' />
+          <StickyNav>
+            <div className='site-container u-padding-Vm'>
+              <ul className='u-pull-start u-margin-Txxs'>
+                <li className='sticky-nav__item'>
+                  <a href='#overview' du-scrollspy ng-gc-smooth-scroll id='track-sticky-nav-overview' className='sticky-nav__link'>
```
Comment (jamesshedden) [inline app/pages/features/features.de.js]: `du-scrollspy ng-gc-smooth-scroll` not needed here or on other elements, leftovers from old Angular-based setup

**Comment by jamesshedden** [comment]
@Ken-Lo Looks good  - ignore my first couple of (deleted) comments, was commenting on the removed code!

Just some remaining attributes from our old setup that have no effect, fine apart from that - let me know if doesn't make sense :+1:

**Comment by Ken-Lo** [comment]
@jamesshedden Nice one, thanks! Have got rid of these on the Pro page as well :)

**Pull request merged**